src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -298,5 +298,11 @@ public interface IDurableOrchestrationClient
         /// <returns>A task that completes when the orchestration is started. The task contains the instance id of the started
         /// orchestratation instance.</returns>
         Task<string> RestartAsync(string instanceId, bool restartWithNewInstanceId = true);
+
+        /// <summary>
+        ///  Steals AppLease if this app doesn't already have it. To use this, must be using the AppLease feature by setting UseAppLease to true in host.json.
+        /// </summary>
+        /// <returns>A task that completes when the steal app message is written to storage.</returns>
```

Comment:
Just for clarity: is writing the steal message to storage the same as completing the process of stealing the lease, or is it just a starting step in that process?

Reply:
We probably want to be explicit in the documentation saying that this kicks off the process to cause this application to steal the app lease.